mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedParams.scala

```diff
@@ -389,4 +389,21 @@ private[ml] trait HasSolver extends Params {
   /** @group getParam */
   final def getSolver: String = $(solver)
 }
+
+/**
+ * Trait for shared param treeDepth (default: 2).
+ */
+private[ml] trait HasTreeDepth extends Params {
```

Comment:
This file is not meant to be changed manually :) Use `SharedParamsCodeGen` instead.